components/TheDataExperienceDefault.vue

```diff
@@ -143,12 +145,14 @@ export default {
     handleRdfEnd() {
       this.progress = false
     },
-    async onUnitFilesUpdate({ inputFiles, error, allFiles }) {
+    onUnitFilesUpdate({ inputFiles, error, allFiles }) {
       this.initState()
       this.inputFiles = inputFiles
       this.allFiles = allFiles
-      if (this.selectedExample.yarrrml) {
```

Comment:
Leaving this is out might be what breaks the the sparql experiences.

Reply:
> The sparql experiences are not working. If you choose tinder, process the files and run any graph, it just says "No relevant data found".
> 
> This is probably related to the deletion of a call to parseYarrrml.

Indeed, this was a mistake I did while merging with the most recent version of master, thanks for catching it!